Good-Bids/goodbids
# [#437] Adjust the price of the Reward to the Winning Bid Subtotal

# Summary

When an Auction Closes, this PR sets the price of the Reward product to the "value" of the Winning Bid. If the Winning Bid is a Free bid, it uses the Subtotal value (instead of 0)

## Issues

* #437 

## Testing Instructions

1. Create a new Product / Auction
2. Set the Price to something that will not match the winning bid.
3. Place a bid on an Auction
4. Wait until it ends
5. Visit WP Admin > Products
6. Find the Reward Product for the Auction
7. Verify the Price is set to the same amount as the Winning Bid.
